specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/ListUsages.json

```diff
@@ -2,58 +2,169 @@
   "parameters": {
     "subscriptionId": "00000000-0000-0000-0000-000000000000",
     "location": "eastus",
-    "api-version": "2018-11-19"
+    "api-version": "2019-05-01"
```

Comment:
If the api-version is this, why are you adding this file in 2018-11-19/examples folder?

Reply:
@sarangan12 , good catch, I synced this with team, I will create a new PR with folder "2019-05-01", when it checked-in, I will update usages changes in another PR.
Here is the PR for migrate version without content changes, please help sign-off: https://github.com/Azure/azure-rest-api-specs/pull/6220